sample/applicationinsights-angularplugin-sample/src/app/telemetry.service.ts

```diff
@@ -1,22 +1,29 @@
-import { Injectable } from '@angular/core';
+import { Injectable, Injector} from '@angular/core';
 import { Router } from '@angular/router';
-import { AngularPlugin } from '@microsoft/applicationinsights-angularplugin-js';
+import { AngularPlugin, ApplicationinsightsAngularpluginErrorService } from '@microsoft/applicationinsights-angularplugin-js';
 import { ApplicationInsights, ICustomProperties, IDependencyTelemetry, IEventTelemetry, IExceptionTelemetry, IMetricTelemetry, IPageViewTelemetry, ITraceTelemetry } from '@microsoft/applicationinsights-web';
 import { environment } from 'src/environments/environment';
 
 @Injectable()
 export class ApplicationInsightsService {
-    private angularPlugin = new AngularPlugin();
+
+     myinjector = Injector.create({
+        providers: [
+            { provide: ApplicationinsightsAngularpluginErrorService, useClass: ApplicationinsightsAngularpluginErrorService }
+        ]
+      });
+
+    private angularPlugin = new AngularPlugin(this.myinjector);
     private appInsights = new ApplicationInsights({
         config: {
-            connectionString: environment.connectionString,
+            instrumentationKey: environment.instrumentationKey,
```

Comment:
We should keep this as the connectionString as the plan is to deprecate only using the `instrumentationKey`